### PR TITLE
Added a scaling speed variable to adjust the speed that hands scale

### PR DESCRIPTION
**Pull request by MattGrayUL**
## Summary

_Summary of the purpose of this merge request._

## Contributor Tasks

_These tasks are for the merge request creator to tick off when creating a merge request._

- [ ] Pair review with a member of the QA team.
- [ ] Add any release testing considerations to the MR for the next release.
- [x] Check any relevant CHANGELOG files have been updated.
- [x] Ensure documentation requirements are met e.g., public API is commented.
- [x] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [x] Add any relevant labels such as `breaking` to this MR.
- [ ] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [ ] Code reviewed.
- [ ] Non-code assets e.g. Unity assets/scenes reviewed.
- [ ] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [ ] Checked and agree with release testing considerations added to MR for the next release.

## Closes JIRA Issue

Closes UNITY-880

### CODE REVIEW

**Comment by MaxPalmer-UH** [comment]
Max is now reviewing this.

**Comment by MaxPalmer-UH** [comment]
Are you also updating the docs for the hand binder?

File: Packages/Tracking/Hands/Runtime/Scripts/HandBinder/HandBinder.cs [hunk 1]
```diff
@@ -61,6 +61,12 @@ public class HandBinder : HandModelBase
         /// </summary>
         public bool SetModelScale = true;
 
+        /// <summary>
+        /// A multiplier to the base speed of the lerp when scaling hands
+        /// </summary>
+        [Tooltip("A multiplier used to adjust the default interpolation of hand scaling")]
+        public float scalingSpeed = 1;
```
Comment (MaxPalmer-UH) [inline Packages/Tracking/Hands/Runtime/Scripts/HandBinder/HandBinder.cs:68]: This uses a different capitalization to all the other inspector properties.

**Comment by MaxPalmer-UH** [comment]
@MattGrayUL this isn't behaving the way I'd expect - I'm going to do some debugging.

**Comment by MattGrayUL** [comment]
> Are you also updating the docs for the hand binder?

I have a docs branch with an additional note for the scaling speed :)

**Comment by MaxPalmer-UH** [comment]
Happy with testing this. Used a tiny hand and then adjusted the factors to the range. Note, it only changes the 'width' of the fingers / thinness of the hand. Also be careful that you are testing the correct hand when setting the factor!

**Review by MaxPalmer-UH — APPROVED** [review]
Happy with testing this. Used a tiny hand and then adjusted the factors to the range. Note, it only changes the 'width' of the fingers / thinness of the hand. Also be careful that you are testing the correct hand when setting the factor!

**Pull request merged**